assets/faq.css

```diff
@@ -183,6 +183,13 @@ code {
 }
 
 @media screen and (max-width: 700px) {
+
+  main div:first-child {
```

Comment:
```suggestion
  main div:first-child {
```

Reply:
I don't see any change here?

Reply:
Oh, it's just a formating change (remove the line break).